src/indicator_cumulative_number.Rmd

```diff
@@ -69,3 +72,70 @@ About point 7 of [#21](https://github.com/trias-project/pipeline/issues/21), ple
 
 see GitHub issue [#20](https://github.com/trias-project/pipeline/issues/20)
 
+```{r Determine presence per species(key) per year}
+
+unique_df <- data %>%
+ filter(!is.na(startDate)) %>%
+ distinct(key, startDate, endDate)
+
+# Create yearslist using startDate and endDate 
+counts_ias <- data.frame(year = c(min(unique_df$startDate, na.rm = TRUE):
+                                 max(unique_df$endDate, na.rm = TRUE))) %>%
+ as_tibble()
+
+count_presence <- function(year, df) {
+ df %>% filter(startDate <= year & endDate >= year) %>%
+   nrow() %>% as_tibble() %>% rename(counts = value)
+}
+
+counts_ias %<>%
+ rowwise() %>%
+ do_(interp(~ count_presence(year = .$year, df = unique_df))) %>%
+ bind_cols(counts_ias) %>% select(year, everything())
```

Comment:
increase indendation to clarify the code structure with piping

Reply:
done